koku/koku/tests_cache.py

```diff
@@ -75,6 +75,16 @@ def test_invalidate_view_cache_for_tenant_and_cache_key(self):
         self.assertIsNone(self.cache.get(key_to_clear))
         self.assertIsNotNone(self.cache.get(remaining_key))
 
+    @override_settings(
+        CACHES={
+            "default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache", "LOCATION": "worker_cache_table"}
```

Comment:
```suggestion
            "default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}
```
The DummyCache doesn't set a location in the settings.py, so don't include one here.